#------------- Rocket parameters --------------------
#
# Defines parameters specific to the rocket:
#   - General dimensions of the components
#   - Rocket engine specifications
#   - Targeted apogee
#
# ----------------------------------------------------
stages: 4
diameters:
  - 0
  - 0.156 
  - 0.156 
  - 0.135
stage_z: 
  - 0 
  - 0.242 
  - 4.11 
  - 4.16
fin_n: 3
fin_top_offset: 3.83 
fin_span: 0.2
fin_root_chord: 0.28 # base length
fin_tip_chord: 0.125
fin_sweep: 0.107
fin_thickness: 0.004
lug_n: 2
lug_S: 5.7e-4 
dry_mass: 36.709
dry_CM: 2.14
dry_I:
  - 52
  - 52
  - 2
propellant_mass: 3.693 
propellant_CM: 3
Isp: 213
maxThrust:
  - 100
  - 100
  - 2000
minThrust: 
  - -100
  - -100
  - 0
minTorque: -50
maxTorque: 50
Cd:
  - 2
  - 2
  - 0.33
throttling: False
input_delay: 34.0e-3 # Mean delay between external actions and its reception
output_delay: 80.0e-3 # Mean delay between output command and its effect
